views/index.hbs

```diff
@@ -15,12 +15,12 @@
 
 			<label aria-describedby="text-box-info" for="freeText" class="o-forms__label">Some text to translate</label>
 			<textarea class="o-forms__textarea" id="freeText"></textarea>
-			
+
 			<label aria-describedby="text-box-info" for="langSelect" class="o-forms__label">Translate to</label>
 			<select id="langSelect" class="o-forms__select">
-				<option value="" disabled selected>Select a language</div>
+				<option value="" disabled>Select a language</div>
```

Comment:
The whole option can be removed now there's a default language

Reply:
done